File: clinking-web/src/store/Shop.ts
// store/shopStore.js
import { defineStore } from "pinia";

interface Shop {
  id: string; // 店铺 ID
  shopId: string; // 店铺的唯一 ID
  shopType: string; // 店铺类型
  shopName: string; // 店铺名称
  shopImage: string; // 店铺图片
  superAdminNickName: string; // 超级管理员昵称
  superAdminAccount: string; // 超级管理员账号
  accessToken: string; // 授权码
  appKey: string; // appKey
  appSecret: string; // appSecret
  totalSale: string; // 总销售额
  status: string; // 状态
  remark: string; // 备注
}

export const useShopStore = defineStore("shop", {
  state: () => ({
    currentShop: {
      shopId: "634418212510386",
      shopName: "CharmHop",
      followers: 99999,
      shopImage:
        "https://img.cdnfe.com/supplier-public-tag/1eac59b83b/f1d28cc8-08db-4a2c-a84c-9d8c28f88b47_300x300.png",
    },
  }),
  actions: {
    setCurrentShop(shop) {
      this.currentShop = shop;
    },
  },
});

File: clinking-server/src/main/java/com/ch/clinking/entity/Shop.java
package com.ch.clinking.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.ToString;

@Data
@ToString
@TableName("shop")
public class Shop {

    @TableId(type = IdType.AUTO)
    private Integer id;

    //店铺ID为创建者的手机号码
    private String shopId;
    //商户类型
    private String shopType;
    //店铺名称
    private String shopName;
    //店铺头像
    private String shopImage;
    //超级管理员昵称
    private String superAdminNickName;
    //超级管理员用户名（账号）
    private String superAdminAccount;

    private String accessToken;

    private String appKey;

    private String appSecret;

    private int totalSale;

    private int status;

    private String remark;
}
